libs/util/httpRequest.js

```diff
@@ -92,19 +79,14 @@ function io(options, controller) {
                     return null;
                 });
             } else {
-                return response.text().then(function (responseBody) {
-                    throw new FetchrError(
-                        options,
-                        request,
-                        response,
-                        responseBody
-                    );
+                return response.text().then(function (body) {
```

Comment:
is body an error here? should we call it error instead?

Reply:
Do you mean an `Error` instance or that the body contains an error message? Well, it's a string and it might contain some error message yes.
But I think it makes sense what you said, however, I think renaming to `err` could be misleading since, normally, `err` points to an `Error` instance. It gets more awkward since just a few line below, we have another `err` name, and that one is a true error.
So I think renaming to `message` could be a good option (since this is the name of `FetchrError` param as well). What do you think?

Reply:
message makes sense to me